jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/internal/HttpChannelState.java

```diff
@@ -1185,7 +1184,7 @@ public void setStatus(int code)
         }
 
         @Override
-        public HttpFields.Mutable getHeaders()
+        public ResponseHttpFields getHeaders()
```

Comment:
Why did this need to be changed?
I cannot see any unique methods that only ResponseHttpFields implement

Reply:
This was for the previous implementation that did freeze/thaw.  It is no longer needed so I will revert.